library/alloc/src/collections/btree/map.rs

```diff
@@ -122,13 +124,14 @@ use UnderflowResult::*;
 /// *stat += random_stat_buff();
 /// ```
 #[stable(feature = "rust1", since = "1.0.0")]
-pub struct BTreeMap<K, V> {
+pub struct BTreeMap<K, V, #[unstable(feature = "allocator_api", issue = "32838")] #[unstable(feature = "allocator_api", issue = "32838")] A: AllocRef = Global> {
```

Comment:
You added `#[unstable]` twice

```suggestion
pub struct BTreeMap<K, V, #[unstable(feature = "allocator_api", issue = "32838")] A: AllocRef = Global> {
```